vortex-datafusion/src/persistent/opener.rs

```diff
@@ -33,35 +28,11 @@ impl VortexFileOpener {
     pub fn new(
         ctx: ContextRef,
         object_store: Arc<dyn ObjectStore>,
-        projection: Option<FieldNames>,
-        predicate: Option<Arc<dyn PhysicalExpr>>,
+        projection: Arc<dyn VortexExpr>,
+        filter: Option<Arc<dyn VortexExpr>>,
         file_layout_cache: FileLayoutCache,
-        file_schema: SchemaRef,
         projected_arrow_schema: SchemaRef,
     ) -> VortexResult<Self> {
-        let dtype = DType::from_arrow(file_schema);
-        let filter = predicate
-            .as_ref()
-            // If we cannot convert an expr to a vortex expr, we run no filter, since datafusion
-            // will rerun the filter expression anyway.
-            .map(|expr| {
-                // This splits expressions into conjunctions and converts them to vortex expressions.
-                // Any inconvertible expressions are dropped since true /\ a == a.
-                let expr = split_conjunction(expr)
-                    .into_iter()
-                    .filter_map(|e| convert_expr_to_vortex(e.clone()).ok())
-                    .reduce(and)
-                    .unwrap_or_else(|| lit(true));
```

Comment:
Making sure we get an empty `Expr` here. Having a `lit(true)` predicate here will make statistics `Inexact` while making no other difference.